src/site/content/en/blog/optimize-cls/index.md

```diff
@@ -104,13 +185,13 @@ This means if we know one of the dimensions, the other can be determined. For a
 
 Knowing the aspect ratio allows the browser to calculate and reserve sufficient space for the height and associated area.
 
-### Modern best practice
+#### Modern best practice
 
 Modern browsers now set the default aspect ratio of images based on an image's width and height attributes so it's valuable to set them to prevent layout shifts. Thanks to the CSS Working Group, developers just need to set `width` and `height` as normal:
```

Comment:
Kind of odd to mention CSS WG here. I'd drop the mention of the WG and focus on the solution itself.

Reply:
Was existing text but OK to change this:

```suggestion
Modern browsers now set the default aspect ratio of images based on an image's `width` and `height` attributes so developers just need to set these, and include the above CSS, to prevent layout shifts:
```